docs/developer-guide/testing.md

```diff
@@ -69,6 +69,16 @@ Other useful constructs are the following:
 | `__goblint_check(1); // reachable` | Checks that the line is reachable according <br> to Goblint results (soundness). |
 | `__goblint_check(0); // NOWARN (unreachable)` | Checks that the line is unreachable (precision). |
 
+#### Meta
+Comments at the end of lines can also indicate metaproperties:
+
+| Annotation | Expected result/comment |
+| ---------- | ----- |
+| `NOCRASH` | No analyzer crash |
+| `FIXPOINT` | No fixpoint error |
+| `NOTIMEOUT` | Analyer terminates |
+| `CRAM` | Automatic checks are only in corresponding Cram test |
+
```

Comment:
I think here we should explain that these annotations only document intention of the test (except for `CRAM`) and any of these issues happening in a test not annotated with this will still cause the test to fail.

So  it is clear that if the analysis, e.g., does not reach a fixpoint for some tests, this causes the test to fail for sure.